en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -11,7 +66,86 @@ We will only consider merging post-beta breaking API changes if they are necessa
+
+- silverstripe/asset-admin [![CI](https://github.com/silverstripe/silverstripe-asset-admin/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-asset-admin/actions/workflows/ci.yml)
+- silverstripe/blog [![CI](https://github.com/silverstripe/silverstripe-blog/actions/workflows/ci.yml/badge.svg?branch=4)](https://github.com/silverstripe/silverstripe-blog/actions/workflows/ci.yml)
+- silverstripe/graphql [![CI](https://github.com/silverstripe/silverstripe-graphql/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/silverstripe-graphql/actions/workflows/ci.yml)
+- silverstripe/elemental-userforms [![CI](https://github.com/dnadesign/silverstripe-elemental-userforms/actions/workflows/ci.yml/badge.svg?branch=4)](https://github.com/dnadesign/silverstripe-elemental-userforms/actions/workflows/ci.yml)
+- silverstripe/versionned-admin [![CI](https://github.com/silverstripe/silverstripe-versioned-admin/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-versioned-admin/actions/workflows/ci.yml)
+- silverstripe/campaign-admin [![CI](https://github.com/silverstripe/silverstripe-campaign-admin/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-campaign-admin/actions/workflows/ci.yml)
+- silverstripe/cms [![CI](https://github.com/silverstripe/silverstripe-cms/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/silverstripe-cms/actions/workflows/ci.yml)
+- silverstripe/elemental-banner-block [![CI](https://github.com/silverstripe/silverstripe-elemental-bannerblock/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/silverstripe-elemental-bannerblock/actions/workflows/ci.yml)
+- silverstripe/session-manager [![CI](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/silverstripe-session-manager/actions/workflows/ci.yml)
+- silverstripe/recipe-ccl [![CI](https://github.com/silverstripe/recipe-ccl/actions/workflows/ci.yml/badge.svg?branch=2)](https://github.com/silverstripe/recipe-ccl/actions/workflows/ci.yml)
+- silverstripe/recipe-content-blocks [![CI](https://github.com/silverstripe/recipe-content-blocks/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/recipe-content-blocks/actions/workflows/ci.yml)
+- silverstripe/recipe-core [![CI](https://github.com/silverstripe/recipe-core/actions/workflows/ci.yml/badge.svg?branch=5)](https://github.com/silverstripe/recipe-core/actions/workflows/ci.yml)
+- silverstripe/staticpublishqueue [![CI](https://github.com/silverstripe/silverstripe-registry/actions/workflows/ci.yml/badge.svg?branch=3)](https://github.com/silverstripe/silverstripe-registry/actions/workflows/ci.yml)
+
+#### Other issues
+
+`tractorcow/silverstripe-fluent` is temporarily being release under `silverstripe/silverstripe-fluent`.
```

Comment:
```suggestion
The CMS 5 compatible version of `tractorcow/silverstripe-fluent` is temporarily being released under `silverstripe/silverstripe-fluent`.
```
We should probably give _some_ explanation for this, and say whether we expect this to change or not when we go stable, or for the RC.

Reply:
I'll use your proposed wording and link off to https://github.com/tractorcow-farm/silverstripe-fluent/issues/776

I didn't want to have a big explanation here because I was afraid this would sound "defensive" and give the wrong impression to people.